static/styles/settings.scss

```diff
@@ -1004,6 +1004,16 @@ input[type=checkbox].inline-block {
     float: right;
 }
 
+.flex-container lab {
```

Comment:
Can you explain why you felt it was necessary to make a new `.flex-container` class? You could've applied the same children selectors to the `.control-group` class, but if there was a particular reason you didn't do so, please elaborate.

Reply:
.flex-container class was added just to enhance readability about "flexbox" being used.
But it obviously adds an unnecessary class.
The only motive was to make the code readable and easy to debug in the future.

Reply:
Yeah, would prefer if you didn't use vague names like `.flex-container` or add extra selectors. Also this is an invalid selector because there's no lab element, only a lab class.